build.gradle

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    testImplementation 'org.jetbrains.kotlin:kotlin-test:1.5.31'
+    testImplementation 'org.jetbrains.kotlin:kotlin-test:1.6.0'
```

Comment:
what made you decide to update this? just curious.

Reply:
Intellij alerted me to update the `kotlin-test` version from 1.5.31 to 1.6.0

Reply:
get in the habit of keeping version updates and the like as separate standalone PRs. You're reviewers will thank you 😄

Reply:
Copy that, will do! I'm guessing the importance of doing this is so the updates don't introduce any unexpected behavior to existing code and to what you're trying to build for the feature? By having a separate standalone PR, the effects are kind of contained?